tutorials/preprocessing/plot_35_artifact_correction_regression.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-artifact-regression:
+
+Repairing artifacts with regression
+===================================
+
+This tutorial covers removal of artifacts using regression as in
+:footcite:`GrattonEtAl1983`.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
```

Comment:
I would remove the TOC because this document has basically just two sections. If you want to keep it, I'd create a new section for the description of the preparatory steps (at the same level as `# Removing artifacts by regression`) to improve the structure.